docs/modules/installation.rst

```diff
@@ -0,0 +1,117 @@
+Installing Protons
+==================
+
+Follow these instructions to install ``protons``, and its dependencies.
+If you run into any issues, please `raise an issue on our Github page`_.
+
+Install using conda
+-------------------
+
+The recommended way to install ``protons`` will be through conda.
+You can obtain conda by installing the Anaconda_ python distribution.
+It will provide a wide suite of scientific python packages, including some of the dependencies in ``protons``.
+Additionally it provides the very powerful ``conda`` python environment and package manager.
+For instructions on how to use conda, please `see this page`_ to get you started.
+
+As part of the first release, we will start building conda packages for ``protons``.
+You will be able to install ``protons`` using the following conda command.
+
+.. code-block:: bash
+
+  conda install -c omnia protons
+
+This will install the latest version of ``protons`` from the Omnia_ channel.
+
+.. _Omnia: http://www.omnia.md/
+.. _Anaconda: https://www.continuum.io/why-anaconda
+.. _see this page: http://conda.pydata.org/docs/get-started.html
+.. Note::
+
+   Currently, no conda packages are being built.
+   You can install dependencies through conda, but the package itself needs to be installed using ``setup.py`` .
+   See below for instructions.
+
+Install using setup.py
+----------------------
+
+Currently, the only way to install protons is by using the ``setup.py`` script that is provided with the package.
+
+Download a copy of the repository, and cd into it.
+
+.. code-block:: bash
+
+    git clone https://github.com/choderalab/constph-openmm.git
+    cd constph-openmm
+
+Them, use the command
+
+.. code-block:: bash
+
+    python setup.py install
+
+to install the package. The ``setup.py`` installation does not automatically check for requirements.
+Make sure to install the dependencies separately using ``pip``, or ``conda``. See below for instructions.
+
+Requirements
+------------
+
+The ``protons`` package has the following requirements:
+
+    - python 2.7, 3.4 or 3.5
+    - openmm_ >=7.0.1
+    - numpy_ >=1.10
+    - scipy_ >=0.17.0
+    - openmmtools_ >=0.7.5
+    - pymbar_
+    - openmoltools_ >=0.7.0
+    - ambermini_
+    - joblib_
+    - lxml_
+
+These should be available from the `omnia` conda channel.
+The following example commands create a new conda environment called "protons", with all dependencies installed.
+
+.. code-block:: bash
+
+    # Point conda to the omnia channel as package source
+    conda config --add channels omnia
+    # Create a new environment called "protons", with all dependencies included
```

Comment:
Good point about the requirements file. I can set one of those up and put it in the docs. I mainly wanted to provide this as a one liner to get people started, though a file would be nicer.